lib/datasource/remote/model/token_model.dart

```diff
@@ -113,8 +117,37 @@ class TokenModel extends Equatable {
   @override
   List<Object?> get props => [chainName, contract, symbol];
 
```

Comment:
Why was this changed to be static?

Reply:
This is the way you make a factory which can return a nullable value.